README.md

```diff
@@ -1 +1,9 @@
-# hello-world
\ No newline at end of file
+# hello-world
+
+Top 5 Rappers of 2020
+
+1. JCole
+2. Travis Scott
+3. Drake
+4. Big Sean
+5. Meg 
```

Comment:
We may need to replace Meg in this case :/

Reply:
Yea, I agree. I don't think she's going to make the cut.